packages/teams-js/src/private/copilot.ts

```diff
@@ -0,0 +1,22 @@
+import { ensureInitialized } from '../internal/internalAPIs';
+import { runtime } from '../public/runtime';
+
+/**
+ * @beta
+ * User information required by specific apps
+ */
+export namespace copilot {
+  export namespace license {
+    /**
+     * This function is called by M365Chat app.
+     * @returns true if a user had M365Chat license and false otherwise
+     *
+     * @throws Error if {@linkcode app.initialize} has not successfully completed
+     *
+     * @beta
+     */
+    export function isSupported(): boolean {
```

Comment:
If the m365ChatLicenseInfo were changed to return an enum, then I suggest adding a separate function in the `copilot.license` subcapability to return the value of that enum, in addition to `isSupported`.